crypto/fipsmodule/aes/mode_wrappers.c

```diff
@@ -142,4 +142,11 @@ int aes_hw_xts_cipher(const uint8_t *in, uint8_t *out, size_t length,
   }
   return 1;
 }
-#endif
+
+#if defined(OPENSSL_X86_64)
+int CRYPTO_is_xts_AESNI_capable(void) {
```

Comment:
What's the motivation for defining this function here? Do we even need it now that it only depends on `CRYPTO_is_AESNI_capable()`?

Reply:
What I recall is that because `CRYPTO_is_AESNI_capable()` is inlined, I couldn't make it directly external [commits 1, 2], I got the same linker error that `OPENSSL_ia32cap_P` is not found in the build. I believe it can be removed once the code moves out of `libdecrepit` into `libcrypto`.
[1] https://github.com/awslabs/aws-lc/pull/429/commits/f4b9931109452b696660402445efc4ceed16b6f8
[2] https://github.com/awslabs/aws-lc/pull/429/commits/b87cf55d2eb860aa347c6549dced83c8853ab69b